tools/syncer.md

```diff
@@ -66,7 +66,7 @@ Usage of syncer:
   -log-rotate string
         to specify the log file rotating cycle, hour/day (default "day")
   -max-retry int
-        to specify maximum retry times during network interruption (default 100)
+        to specify maximum retry times of the SQL request when an error occurs due to network interruption or other reasons (default 100)
```

Comment:
```suggestion
        specifies the maximum times an SQL statement should be retried. One common cause of statement retries is network interruption (default 100)
```
I think it's better not to say "or other reasons" and not specify. Here we can say it is a common cause, but imply it is not the only one.  I assume it might be transaction conflicts with retry disabled that is another, but that is complex to explain here.